src/schema/v1.json

```diff
@@ -349,6 +350,12 @@
          "title":"Metal",
          "$ref": "#/definitions/artifact"
         },
+       "metal4k": {
+         "$id":"#/properties/images/properties/metal4k",
+         "type":"object",
+         "title":"Metal4k",
```

Comment:
Maybe something like "Metal (4K native)" for the title?

Reply:
makes sense...done